apps/tx-builder/src/utils.ts

```diff
@@ -118,21 +118,3 @@ export const encodeToHexData = (contractMethod: ContractMethod | undefined, cont
     }
   }
 };
-
-export const getTxDescription = (contractMethod: ContractMethod | undefined, contractFieldsValues: any) => {
```

Comment:
we do not need this function because now we are showing a simple description for each transaction